core/src/database/PostgreSQLBackend.cpp

```diff
@@ -70,5 +81,18 @@ namespace ledger {
                                             soci::session &session) {
             throw make_exception(api::ErrorCode::IMPLEMENTATION_IS_MISSING, "Change of password is not handled with PostgreSQL backend.");
         }
+
+        void PostgreSQLBackend::initSSLLibraries() {
+#ifdef SSL_SUPPORT
+            std::once_flag sslFlag;
```

Comment:
shouldn't sslFlag be static, otherwise it is new value each call, right?

Reply:
The link you pasted even puts that as a global 😨